### DISCUSSION TITLE
What color theme is used for the screenshots?

### DISCUSSION THREAD

**Discussion by vdawg-git** [smolck/uivonim#625] [Q&A] [answered]
It looks so nice. 

<img src="https://user-images.githubusercontent.com/46855713/132102617-8a85965c-8ad7-4cc6-bd34-c63db2197179.png" />

**Comment by smolck** [comment, marked as answer]
It’s a custom color theme that’s bundled with uivonim, see https://github.com/smolck/uivonim/blob/master/runtime/colors/veonim.vim

This reminds me, maybe I will turn it into its own separate theme, I do quite like it myself ;)

**Reply by vdawg-git** [reply]
Nice, thank you :)
